app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/KeyGeneratorAdapter.java

```diff
@@ -75,4 +73,14 @@ public KeyPair generateRSAKeyPair(final String name) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public KeyPair generateEd25519KeyPair(final String name) throws KeyGenerationException {
+        final Ed25519Key publicKey = new Ed25519Key(name, "foo");
```

Comment:
I will fix it in this PR, I'll soon it push (was a bit more complicated to implement then expected 😅)